### DISCUSSION TITLE
Migrating from Wordpress and help required

### DISCUSSION THREAD

**Discussion by Grampanda** [gohugo-ananke/ananke#631] [3 - Ideas]
I'm migrating (myself, not a website) from Wordpress and I am not making a lot of progress with Hugo/ananke.

I have followed the "Getting Started" pages to install Hugo on my MacBook and now I have one Post visible on "My New Hugo Site" but there is no menu to view any pages (or do you call these articles) .  I have tried adding a directory QuickStart/content/pages and QuickStart/content/articles, added a page there but I do not see these in "My New Hugo Site".

From the documentation it appears Ananke does not support pages/articles but when I look at the Ananke Demo site I see a Menu and Articles.

So, please tell me the next steps from having "My first Post" to something like the Ananke Demo site?

**Comment by budparr** [comment]
You'd be better off asking the Hugo community here: https://discourse.gohugo.io

**Reply by Grampanda** [reply]
OK, thanks.  Will do.
